controller/router.js

```diff
@@ -95,6 +97,16 @@ module.exports = [
                         .number()
                         .integer()
                         .positive()
+                },
+                query: {
+                    fromDate: Joi
+                       .date()
+                       .format('YYYY-MM-DD')
+                       .default(moment().startOf('day').toDate()),
+                    toDate: Joi
+                       .date()
+                       .format('YYYY-MM-DD')
+                       .default((moment().add(addSevenDays, 'days')).toDate())
```

Comment:
exactly, and here x is one week ago and (x + 7) is today :smile: 
Calculating compliance 1 week in the future doesn't make sense.